packages/bpk-docs/src/layouts/DefaultLayout/default-layout.scss

```diff
@@ -20,7 +20,21 @@
 
 .bpkdocs-default-layout {
   &__container {
-    background-color: $bpk-color-gray-100;
+    position: absolute;
+    display: flex;
+    width: 100%;
+    min-height: 100%;
+    flex-direction: column;
+  }
+
+  &__content-container {
+    display: flex;
+    flex-direction: column;
+    flex: 1 0 auto;
+
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
This is a hack that is required for IE 11 only. It is very hacky, but the worst case scenario is that we change the footer size, and the alignment goes slightly wrong in IE 11.

Reply:
can we add this comment in the code? explaining why this is necessary?

Reply:
Not a fan of this sort of hack, where there's a line that only one browser can see.. If we do go down this road I think we should at least make it a mixin, so it's a bit clearer:

```
@include internet-explorer-only {
  // Code here
}
```